kaztron/cog/dice.py

```diff
@@ -84,6 +86,25 @@ def __init__(self, bot):
         await self.bot.say("[{}]\n**Sum:** {:d}".format(' '.join(rolls_str), total))
         logger.info("Rolled FATE dice: {!r} (sum={})".format(rolls_str, total))
 
+    # strip_list = map(str.strip, lines)
+    @commands.command(pass_context=True, ignore_extra=False, no_pm=False)
+    @in_channels(ch_allowed_list)
+    async def choose(self, ctx, *, choices: str):
+        "Need some help making a decision? Just give the .choose command two or more options(separated by ,) and it will choose for you!"
+        choices = list(map(str.strip, choices.split(",")))
+        if "" in choices:
+            logger.warning("choose(): argument empty")
+            await self.bot.say("I cannot decide if there's an empty choice.")
```

Comment:
UX - I would suggest just excluding/filtering out empty choices from the list instead of erroring - it's not really a problem if someone typos two commas or something, if there's at least 2 choices otherwise: `.choice flamingo,, grapefruit`